member-service/src/main/java/com/hedvig/memberservice/identity/repository/IdentityEntity.kt

```diff
@@ -0,0 +1,38 @@
+package com.hedvig.memberservice.identity.repository
+
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.UpdateTimestamp
+import java.time.Instant
+import javax.persistence.Embedded
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.Id
+
+@Entity
+class IdentityEntity(
+    @Id
+    val memberId: Long,
```

Comment:
Not entirely sure about this but I think we prefer having memberId as a String but incorrectly made it into a Long in member-service. Don't know for sure though
```suggestion
    val memberId: String,
```